npm/oxlint/bin/oxlint

```diff
@@ -1,19 +1,3 @@
 #!/usr/bin/env node
 
-// This script is run as CommonJS, so use dynamic import to load ESM module `dist/cli.js`.
-// Even if runtime supports require(ESM), can't use `require` here, because `dist/cli.js`
-// uses top-level `await`.
-//
-// `dist/cli.js` uses require(ESM) internally, but only on path where `--js-plugins`
-// CLI option is used. So we still support runtimes without require(ESM) for users who aren't
-// using experimental options.
-import("../dist/cli.js").catch(onError);
-
-function onError(err) {
```

Comment:
is it ok to drop this onError handling?

Reply:
Yes. It was only introduced in #13723 as a workaround for the module being CommonJS. Now any errors will just output a stack trace in the usual way - we don't need to print the stack trace manually.